.vscode/launch.json

```diff
@@ -4,25 +4,36 @@
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
+
         {
-            "name": "debug cedana-cli run",
+            "name": "sudo debug cedana-cli run",
             "type": "go",
             "request": "launch",
             "mode": "debug",
-            "program": "${fileDirname}",
+            "program": "${workspaceFolder}",
+            "console": "integratedTerminal",
+            "args": ["run", "test/integration/jobs/job.yml"],
+
+        },
+        {
+            "name": "sudo debug cedana-cli whisper restore",
+            "type": "go",
+            "request": "launch",
+            "mode": "debug",
+            "program": "${workspaceFolder}",
             "console": "integratedTerminal",
-            "args": ["run", "caltech_finetune_test.yml"],
-        
+            "args": ["whisper", "restore", "-j", "cjigihavbi2i8gns57g0"],
```

Comment:
add these args as an input param. Take a look at the launch.json in the cedana repo to see how

Reply:
Ah yes good catch thanks!